examples/goldilocks_models/find_models/traj_opt_given_weigths.cc

```diff
@@ -236,7 +236,12 @@ void extractResult(VectorXd& w_sol,
       return;
     }
   }
+
   cout << "storing result...\n";
+  // store solve time for Traj Opt
+  if (n_rerun == 0){
+      writeCSV(directory + prefix + string("solve_time.csv"), elapsed.count()*MatrixXd::Ones(1, 1));
+  }
```

Comment:
Should remove this if it's for testing purposes

Reply:
Removed